controllers/pod_controller.go

```diff
@@ -46,10 +48,15 @@ type PodReconciler struct {
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, err error) {
-	if !r.IgnoreNamespaces.In(req.Namespace) {
-		log.FromContext(ctx).V(1).Info("Triggering reconciliation")
-		r.Triggerer.Trigger()
+	for _, regex := range r.IgnoreNamespaces {
```

Comment:
nit: it looks like `isNamespaceIgnored` can be moved o a common package and used here too